src/oc/hoc.c

```diff
@@ -109,13 +109,15 @@ void pr_profile(void) {
 	}
 }
 #else
-void start_profile(i) int i; {}
-void add_profile(i) int i; {}
+void start_profile(i) int i;
+{}
```

Comment:
I find this format surprising.  The original seems better or at least a single '}' on line 113

Reply:
I wonder if this is because of `K&R style definition`. Seems like clang-format is assuming `void start_profile(i) int i;` as first statement and `{}` as separate statement?